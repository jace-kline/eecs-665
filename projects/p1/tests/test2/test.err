FATAL [1,52]: Illegal character \
FATAL [1,55]: Illegal character .
FATAL [1,56]: Illegal character .
FATAL [1,57]: Illegal character .
FATAL [1,77]: Illegal character ?
FATAL [1,78]: Empty character literal
flex scanner jammed
