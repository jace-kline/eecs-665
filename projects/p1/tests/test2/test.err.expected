FATAL [2,1]: Unterminated string literal ignored
FATAL [3,19]: Illegal character ?
FATAL [3,20]: Unterminated string literal ignored